resources/views/join.blade.php

```diff
@@ -45,6 +49,14 @@
       </span>
         </p>
       </div>
+
+      <div class="field">
+        <label class="label">Allergies / Food Preferences</label>
+        <div class="control">
+          <textarea id="allergies" class="input" rows="5" placeholder="List All Allergies" name="allergies" onchange="hideErrors()"></textarea>
```

Comment:
PLease use this change for the textarea (info [HERE](https://bulma.io/documentation/form/textarea/))
```suggestion
          <textarea id="allergies" class="textarea" rows="5" placeholder="List All Allergies" name="allergies" onchange="hideErrors()"></textarea>
```